test.js

```diff
@@ -101,6 +101,20 @@ test('directories', async t => {
 	t.false(fs.existsSync(321));
 });
 
+test('tons of files', async t => {
+	const paths = [];
+	for (let i = 0; i < 10000; i++) {
```

Comment:
Reduce this to 5000. 10000 is too slow. Also move the number into a constant so it's easily modified, and reused on line 113.